### PR TITLE
Drop support for CentOS 7

### PR DESCRIPTION
**Pull request by jaimesouza**
## Description

Drop support for CentOS 7.

## How was the code tested?

The code was tested locally with LXD containers running Ubuntu 22.04.

## Checklist

- [x] I am the author of these changes, or I have the rights to submit them.
- [ ] I have added the relevant changes to the README and/or documentation.
- [x] I have self reviewed my own code.
- [ ] All requested changes and/or review comments have been resolved.

### CODE REVIEW

File: dispatch
Comment (NucciTheBoss) [inline dispatch:1]: Similar to my comment https://github.com/omnivector-solutions/slurmctld-operator/pull/18#discussion_r1337269095 on your pull request against the slurmctld operator.

**Review by NucciTheBoss — CHANGES_REQUESTED** [review]
LGTM, just one minor comment.

File: dispatch
Comment (jaimesouza) [reply dispatch:1]: That is right! Thanks @NucciTheBoss!

**Review by NucciTheBoss — APPROVED** [review]
LGTM! Nice work :tada:

**Pull request merged**